pkg/plugin/util/util.go

```diff
@@ -72,6 +72,9 @@ func InsertCode(filename, target, code string) error {
 		return err
 	}
 	idx := strings.Index(string(contents), target)
+	if idx == -1 {
+		return fmt.Errorf("string %s not found in %s", target, string(contents))
+	}
```

Comment:
Could we add a test to cover this one?

Reply:
https://github.com/NikhilSharmaWe/kubebuilder/commit/1d691bbb40e230b240610c880e8b7fe2ce6a2b7f